pom.xml

```diff
@@ -83,6 +83,25 @@
         <overridesfile>${resourcesdir}/extensions-overrides.json</overridesfile>
     </properties>
 
+    <repositories>
+        <repository>
+            <snapshots>
+                <enabled>false</enabled>
+            </snapshots>
+            <id>kogito</id>
+            <name>kogito</name>
+            <url>https://repository.jboss.org/nexus/content/groups/kogito-public/</url>
```

Comment:
What is the reason for adding this repo?

Reply:
this is the staging repo. It will be removed if this is green, and we'll progress with our own build, promoting to Central